data/campaigns/Heir_To_The_Throne/scenarios/05b_Isle_of_the_Damned.cfg

```diff
@@ -341,60 +341,58 @@
             message= _ "Surprise! Searching for magi, and all I get is these foul humans!"
         [/message]
     [/event]
+#endif
 #enddef
 
     [event]
         name=prestart
 
-        #
-        # Determine the contents of the temples:
-        # - on easy, Moremirmu shows up in 1 close temple and the others are empty
-        # - on normal, Moremirmu shows up in 1 close temple and the far one has the revenant
-        # - on hard, Moremirmu and the revenant show up in the 2 close temples (far one is empty)
-        #
-        {RANDOM 1..2}
-        [if]
-            [variable]
-                name=random
-                numerical_equals=1
-            [/variable]
-#ifdef EASY
-            [then]
+        {RANDOM 1..6}
```

Comment:
So, those removed comments are no longer true? I feel like pushing the revenant from the far temple to a close temple could have been a way to make it more difficult though...